docs/background/autoscaling.md

```diff
@@ -0,0 +1,3 @@
+# Autoscaling
+
+Coming soon!
```

Comment:
So my interpretation of [the Divio documentation system](https://documentation.divio.com/introduction/) is that this shouldn't be "background information." It should be "Explanations" on things like "Why should I use Anycloud?" "Why *shouldn't* I use Anycloud?" "What complications do multi-region and/or multi-cloud deployments have?" etc.

I think short essay-level articles on things like that are more useful for end users and closer to the intent of this "quadrant."

Reply:
So the link I shared explicitly calls it "Explanation" not "Background Information." It's possible that they're still feeling out what that quadrant is useful for, themselves. I think "Explanation" is a better fit and more useful in general because it has a better focus on "what questions would the user be asking?" rather than "here's a dump of random crap that we think is relevant background."

Moving on, I don't entirely agree with the assertion that it should be in the homepage. You can have a paragraph or two on what Anycloud is best for, but a deep dive article on *why* it really works better in certain circumstances deserves more attention/clarity, and is a good thing to share out.

Also, these two files are still placeholders, so perhaps we should just delete the whole section until we're ready to put things here?

Reply:
I'm game to remove the section until we're ready to put things here